src/Sql/Sql.Test/Sql.Test.csproj

```diff
@@ -20,6 +30,7 @@
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="3.1.0-preview.2" />
     <PackageReference Include="Microsoft.Azure.Management.OperationalInsights" Version="0.21.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.44.2-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="17.2.0" />
```

Comment:
@MilanBrkicFON 
You should not reference Az.Storage module in test, or it will take a lot of trouble for cross reference, especially when upgrade SDK version.
I have filed an issue for it in: https://github.com/Azure/azure-powershell/issues/11891

For create storage  account or blob container, you can untilize  “Microsoft.Azure.PowerShell.Clients.Storage.Management” . 
There's a PR to update it https://github.com/Azure/azure-powershell-common/pull/230, and I have add comments should add create blob container API in it.